CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
 
+**2.2.0 Release: March 15, 2021**
+
+## Release Notes
+
+- Cleaning up and parallelizing Statcast calls -- @TheCleric, @bdilday, and @schorrm
+
+- Statcast spin rate & spin direction -- @tpoatsy, see docs/statcast_pitcher_spin.md for more info.
+
+- Statcast Batter & Pitcher leaderboards -- @wfordh
+
+- We removed some fairly unnecessary dependencies that were causing a massive amount of dependency bloat -- @TheCleric and @bdilday.
+
+- Various test fixes -- @TheCleric
+
+- Spell-check to playerid lookup -- @tjburch and @mwisnie5
+
+- Plotting bugfix -- @andersonfrailey
+
+- Marcels Example -- @andersonfrailey
+
```

Comment:
This is copied from the README and I think it all relates to the latest release. If not, I'm happy to update/remove the relevant bits.